homeassistant/components/goalzero/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Goal Zero Yeti",
+        "description": "First, you need to download the Goal Zero app: https://www.goalzero.com/product-features/yeti-app/\n\nFollow the instructions to connect your Yeti to your Wifi network. Then get the host ip from your router. DHCP must be set up in your router settings for the device to ensure the host ip does not change. Refer to your router's user manual.",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "name": "Name"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_host": "This is not the Yeti you are looking for",
```

Comment:
Maybe clarify that it's the host input that's incorrect?

Reply:
This was a tortured obiwan kenobi reference. I'll change it if you insist 😉

Reply:
Yes, please clarify it. :+1: